### ISSUE TITLE
ovpn_initpki script : error with touch command

### ISSUE THREAD

**Issue by janvda** [Salvoxia/docker-openvpn-tap#1]
touch command is not working in the ovpn_initpki script.

I am getting on my ubuntu server:

```
root@nuc1:~# docker run -v ovpn-data:/etc/openvpn --rm -it salvoxia/openvpn-tap:0.4 ovpn_initpki
BusyBox v1.36.1 (2023-11-07 18:53:09 UTC) multi-call binary.

Usage: touch [-cham] [-d DATE] [-t DATE] [-r FILE] FILE...

Update mtime of FILEs

        -c      Don't create files
        -h      Don't follow links
        -a      Change only atime
        -m      Change only mtime
        -d DT   Date/time to use
        -t DT   Date/time to use
        -r FILE Use FILE's date/time
root@nuc1:~# 
```

**Comment by janvda** [comment]
It seems to be working when I remove following line (line 19) from [ovpn_initpki](https://github.com/Salvoxia/docker-openvpn-tap/blob/master/bin/ovpn_initpki) script.

```
touch $EASYRSA_VARS_FILE
```

**Comment by Salvoxia** [comment]
Thanks, this should be fixed with https://github.com/Salvoxia/docker-openvpn-tap/commit/8de9107e203d015155b151008c19ce182c7ac0a8 in the [latest](https://github.com/Salvoxia/docker-openvpn-tap/pkgs/container/openvpn-tap/171755503?tag=latest) image. Can you confirm that's fixed for you?

**Comment by janvda** [comment]
Indeed the latest docker hub image fixes the issue.

**Comment by Salvoxia** [comment]
Fixed with [0.5](https://github.com/Salvoxia/docker-openvpn-tap/releases/tag/0.5)
